### DISCUSSION TITLE
AssertionError: MMCV==1.6.1 is used but incompatible. Please install mmcv>=1.3.8, <=1.6.0.

### DISCUSSION THREAD

**Discussion by smahmood00** [open-mmlab/mmpose#1571] [Q&A]
I am using MMPose and have completed the Prerequisites and Installation steps from as mentioned in https://mmpose.readthedocs.io/en/latest/install.html

**Note:** I am using CPU based pytorch and torchvision

I get the following assertion error : 
**AssertionError:** MMCV==1.6.1 is used but incompatible. Please install mmcv>=1.3.8, <=1.6.0.

Your suggestion will be appreciated to solve this issue. Thanks.

**Comment by jin-s13** [comment]
Please refer to 
https://github.com/open-mmlab/mmpose/blob/f7b4b257d1364011966bc9eda9295fdd93684a0e/mmpose/__init__.py#L20
